bmv2/Makefile.am

```diff
@@ -40,7 +43,7 @@ pd_files.ts : $(P4_PATH) $(refdeps)
 	@touch pd_files.tmp
 	PYTHONPATH=$$PYTHONPATH:$(MY_PYTHONPATH) $(P4C_BM) --pd $(builddir)/$(P4_NAME)/ --p4-prefix $(P4_PREFIX) $(P4C_BM_FLAGS) --json $(builddir)/$(P4_JSON_OUTPUT) $(P4_PATH)
```

Comment:
--plugin is provided in P4C_BM_FLAGS

On Wed, Jun 15, 2016 at 1:12 PM, Antonin Bas <notifications@github.com>
wrote:

> In bmv2/Makefile.am
> <https://github.com/p4lang/p4-build/pull/5#discussion_r67236544>:
>
> > @@ -40,7 +43,7 @@ pd_files.ts : $(P4_PATH) $(refdeps)
> >  	@touch pd_files.tmp
> >  	PYTHONPATH=$$PYTHONPATH:$(MY_PYTHONPATH) $(P4C_BM) --pd $(builddir)/$(P4_NAME)/ --p4-prefix $(P4_PREFIX) $(P4C_BM_FLAGS) --json $(builddir)/$(P4_JSON_OUTPUT) $(P4_PATH)
>
> shouldn't there be a --plugin option somewhere?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/p4lang/p4-build/pull/5/files/86bb6121bafc3dda8c3a840c9bd574fee38038ee#r67236544>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFgBcVUi0Xd9mslxD9BDNfe8CRE9fIIPks5qMFyfgaJpZM4Iylfb>
> .
>